docs/res/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
```

Comment:
Nit: What about using `pushd` and `popd` to make sure the caller doesn't end up in a different folder after the script finishes?

Reply:
Agreed.

Reply:
This isn't actually necessary, as everything is done in a subshell, so it won't affect the calling shell's directory.